examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -822,37 +832,37 @@ void systemResetCallback()
 }
 
 void printWifiStatus() {
-#if defined(ARDUINO_WIFI_SHIELD) || defined(WIFI_101)
+#if defined(ARDUINO_WIFI_SHIELD) || defined(WIFI_101) || defined(ESP8266_WIFI)
```

Comment:
I don't think any of these #ifdefs are needed anymore since all supported libraries support the following function calls. If a lib does not support one of these calls in the future, we could use an #ifndef instead. I really want to try to reduce the number of conditional compiler statements in the StandardFirmata variants.